.github/workflows/cicd.yml

```diff
@@ -235,7 +235,17 @@ jobs:
           tox run -e lint --notest
 
       - name: Run unit tests
-        run: tox run -e ${TOX_ENV}
+        id: test
+        run: |
+          status=0
+          tox run -e "${TOX_ENV}" || status=$?
```

Comment:
IIRC `$?` is a feature of any shell, most likely any shell in Microsoft Github Actions, therefore both lines can be condensed to a single line on the second lines' composition:

~~~
tox run -e "${TOX_ENV}"; status=$?
~~~

Thanks for posting a useful comment updating the other thread! @wsanchez